src/Vipps.net/Vipps.net.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;netcoreapp3.1;netstandard2.0;net48</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
isn't this what allows merchants to use older versions of .net and the sdk?

Reply:
No idea tbh. I am way out of my depth here. First time touching a .net project 🥵

Reply:
Reverting